topic-controller/src/test/java/io/strimzi/controller/topic/ControllerAssignedKafkaImplTest.java

```diff
@@ -110,7 +119,9 @@ protected void addJavaArgs(List<String> args) {
     public void changeReplicationFactor_missingExecutable(TestContext context) {
         MockAdminClient adminClient = new MockAdminClient();
         Vertx vertx = Vertx.vertx();
-        Config config = new Config(singletonMap(Config.REASSIGN_VERIFY_INTERVAL_MS.key, "1 seconds"));
+        Map<String, String> map = new HashMap<>(mandatory);
```

Comment:
I think all the tests in this class are doing the same thing with creating `map`, and then creating a `Config` from it. Let's use this an an opportunity to factor out a field which can be used in all those tests.